Dockerfile.windows

```
@@ -152,15 +153,15 @@
 # -----------------------------------------------------------------------------------------
 
 
+# Extremely important. If using this Dockerfile in process isolated containers,
+# the kernel of the host must match the container image, and hence would fail
+# between Windows Server 2016 (aka RS1) and Windows Server 2019 (aka RS5).
+ARG WINDOWS_BASE_IMAGE=mcr.microsoft.com/windows/servercore:1809
+ARG WINDOWS_BASE_IMAGE_TAG=${WINDOWS_BASE_IMAGE_TAG:-1809}
```

These are used via `FROM ${WINDOWS_BASE_IMAGE}:${WINDOWS_BASE_IMAGE_TAG}` below so should probably be the following, right?  (no need to use `:-` because this is `ARG` so it's already overridden by anything specified by the user)

```suggestion
ARG WINDOWS_BASE_IMAGE=mcr.microsoft.com/windows/servercore
ARG WINDOWS_BASE_IMAGE_TAG=1809
```

---

Ah, will have to give it another look. I rebased this PR earlier this week, to see in what shape it was (but need to refresh my memory on the changes ☺️)

Thanks for looking; I'll try to do the "refresh my memory" soon 💜